index.html

```diff
@@ -14,164 +14,197 @@
         <nav>
             <img class="menubutton" src='./images/Union.svg' alt="menubutton">
         </nav>
-    </header>
-    <section class="container mobilepara">
-        <h1>I'm Medini<br>Glad to see<br>you!</h1>
-        <p>I'm a software developer! I can help you
-            build a product, feature or website Look
-            through some of my work and 
-            experience! If you like what you see and
-            have a project you need coded, don't
-            hesitate to contact me.</p>
-        <a href=#>LET'S CONNECT</a>
-        <ul class="social_media_icons">
-            <li><a href=#><img src='./images/twitter.svg' alt="twitter icon"></a></li>
-            <li><a href=#><img src='./images/Linkedin_icon.svg' alt="LinkedIn icon"></a></li>
-            <li><a href=#><img src='./images/m_icon.svg' alt="M icon"></a></li>
-            <li><a href=#><img src='./images/cat_icon.svg' alt="Cat icon"></a></li>
-            <li><a href=#><img src='./images/hand_icon.svg' alt="Hand icon"></a></li>
+        <ul class="header_list_items">
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
         </ul>
+    </header>
+    <section class="container mobilepara grid">
+        <div>
+            <h1 class="h1_mobile">I'm Medini<br>Glad to see<br>you!</h1>
+            <h1 class="h1_desktop">I'm Medini<br>Glad to see you!</h1>
+            <p>I'm a software developer! I can help you
+                build a product, feature or website Look
+                through some of my work and 
+                experience! If you like what you see and
+                have a project you need coded, don't
+                hesitate to contact me.</p>
+            <a href=#>LET'S CONNECT</a>
+            <ul class="social_media_icons">
+                <li><a href=#><img src='./images/twitter.svg' alt="twitter icon"></a></li>
+                <li><a href=#><img src='./images/Linkedin_icon.svg' alt="LinkedIn icon"></a></li>
+                <li><a href=#><img src='./images/m_icon.svg' alt="M icon"></a></li>
+                <li><a href=#><img src='./images/cat_icon.svg' alt="Cat icon"></a></li>
+                <li><a href=#><img src='./images/hand_icon.svg' alt="Hand icon"></a></li>
+            </ul>
+        </div>
     </section>
-    <div class="grid-container">
+    <div class="grid-container grid">
```

Comment:
- [OPTIONAL] Please make sure this section is aligned a per the design, your projects seem to be aligned more to the left.  Kindly make sure your margins and alignments are as in the design. See image below

***The design***
![medD](https://user-images.githubusercontent.com/65852621/135704898-14b5d60c-6195-4b88-8d61-7cd86b83e226.png)

Reply:
Thank you for your review. Since this is optional comment, I will implement in my future work.